Update libgfxd to 7dd7fa1

Dragorn421 opened the pull request:
Update libgfxd to 7dd7fa16227f6d4c336abfda353d01f833334413

Also fixes #5 and an unimportant issue in gfx_ucode

I have had no issue using this "updated" pygfxd :)

Dragorn421 committed:
Update python bindings to libgfxd 7dd7fa1

libgfxd 7dd7fa16227f6d4c336abfda353d01f833334413

Update/add `gfxd_light_callback`, `gfxd_lightsn_callback`
Add binding for `gfxd_foreach_pkt`
Update description for `gfxd_macro_fn`

Dragorn421 committed:
Fix https://github.com/Thar0/pygfxd/issues/5

Dragorn421 committed:
Fix/update `gfx_ucode` structure in py binding

Thar0 commented on pygfxd.py:
```diff
@@ -853,6 +869,24 @@ def gfxd_macro_packets() -> int:
+lgfxd.gfxd_foreach_pkt.restype = c_int
+def gfxd_foreach_pkt(fn: Callable[[], int]) -> int:
+    """
+        int fn(void))
```
```suggestion
    int fn(void);
```
I guess this is just to notate the expected form of `fn`?

Thar0 reviewed (commented):
One thing

Dragorn421 committed:
git subrepo pull (merge) libgfxd

subrepo:
  subdir:   "libgfxd"
  merged:   "7dd7fa1"
upstream:
  origin:   "https://github.com/glankk/libgfxd"
  branch:   "master"
  commit:   "7dd7fa1"
git-subrepo:
  version:  "0.4.5"
  origin:   "git@github.com:ingydotnet/git-subrepo.git"
  commit:   "dbb99be"
